packages/mux-video/src/index.ts

```diff
@@ -380,6 +402,16 @@ class MuxVideoElement extends CustomVideoElement<HTMLVideoElement> implements Pa
     }
   }
 
+  // NOTE: This needs to be public so it is accessible to playback-core
```

Comment:
Is there a need for this method? Wouldn't playback-core triggering a cuepointchange on the media-el be sufficient?